ast/Trees.h

```diff
@@ -716,6 +719,24 @@ TREE(Send) : public Expression {
     virtual std::string showRaw(const core::GlobalState &gs, int tabs = 0);
     virtual std::string nodeName();
 
+    std::pair<int, int> kwArgsRange() const {
+        auto res = std::make_pair<int, int>(numPosArgs, args.size());
```

Comment:
Add a comment here about whether the values returned are inclusive/exclusive in the range they define.